database/seeders/ImmobilisationSeeder.php

```
@@ -13,11 +13,11 @@ class ImmobilisationSeeder extends Seeder
     public function run(): void
     {
         $categories = [
-            AssetCategory::create(['name' => 'Matériel de BTP', 'account_code' => '215100', 'default_depreciation_years' => 5, 'default_method' => DepreciationMethod::LINEAR]),
-            AssetCategory::create(['name' => 'Outillage', 'account_code' => '215400', 'default_depreciation_years' => 3, 'default_method' => DepreciationMethod::LINEAR]),
-            AssetCategory::create(['name' => 'Mobilier de bureau', 'account_code' => '218200', 'default_depreciation_years' => 5, 'default_method' => DepreciationMethod::LINEAR]),
-            AssetCategory::create(['name' => 'Matériel informatique', 'account_code' => '218300', 'default_depreciation_years' => 3, 'default_method' => DepreciationMethod::LINEAR]),
-            AssetCategory::create(['name' => 'Véhicules', 'account_code' => '211000', 'default_depreciation_years' => 4, 'default_method' => DepreciationMethod::LINEAR]),
+            AssetCategory::create(['name' => 'Matériel de BTP', 'account_code' => '215100', 'compte_amortissement' => '281500', 'default_depreciation_years' => 5, 'default_method' => DepreciationMethod::LINEAR]),
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\Immobilisation\AssetCategory::create().


(staticMethod.notFound)